src/routes/SignOut.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+function SignOut() {
+  const navigate = useNavigate();
+
+  const handleSignOut = async () => {
+    try {
+      const response = await fetch(
+        'https://doctalk-r977.onrender.com//users/sign_out',
+        {
+          method: 'DELETE',
+          headers: {
+            'Content-Type': 'application/json',
+            Authorization: localStorage.getItem('token'),
+          },
+        },
+      );
+      localStorage.removeItem('token');
```

Comment:
[OPTIONAL] For security reasons, `localStorage` is not the best option for storing sensitive data, consider using `httpOnly` cookies or session management modules. This is okay for this project, but consider this in future